src/js/tech/html5.js

```diff
@@ -962,6 +962,29 @@ Html5.canControlVolume = function() {
   }
 };
 
+/**
+ * Check if the volume can be muted in this browser/device.
+ * Some devices, e.g. iOS, don't allow changing volume
+ * but permits muting/unmuting.
+ *
+ * @return {bolean}
+ *      - True if volume can be muted
+ *      - False otherwise
+ */
+Html5.canMuteVolume = function() {
```

Comment:
Hmmm... My tests hasn't shown this so far.

Reply:
specifically, given that we're always setting the muted attributed, there may be a discrepancy between it and the muted property

Reply:
Hey @gkatsev, I might be missing something here but I'm not seeing anywhere where {{muted}} attribute would be set on {{Html5.TEST_VID}} element other than this method itself. If you could elaborate a bit more that'd be much appreciated. Thanks!

Reply:
this is probably a minor thing but my concern is that we're setting the attribute on line 905 to always be muted but the muted property is set to the opposite of what it used to be.
So, if you call `canMuteVolume` multiple times, you may end up getting an unexpected answer because of how the property and the attribute interact with the element.

Reply:
Gotcha, thanks.